Formula/m/mise.rb

```diff
@@ -41,14 +41,18 @@ def install
 
     system "cargo", "install", *std_cargo_args
     man1.install "man/man1/mise.1"
-    generate_completions_from_executable(bin/"mise", "completion")
     lib.mkpath
     touch lib/".disable-self-update"
     (share/"fish"/"vendor_conf.d"/"mise-activate.fish").write <<~FISH
       if [ "$MISE_FISH_AUTO_ACTIVATE" != "0" ]
         #{opt_bin}/mise activate fish | source
       end
     FISH
+
+    # Untrusted config path problem, `generate_completions_from_executable` is not usable
+    bash_completion.install "completions/mise.bash" => "mise"
+    fish_completion.install "completions/mise.fish"
+    zsh_completion.install "completions/_mise"
```

Comment:
might worth raising a bug?

Reply:
Once we run `mise`, it will pop up some interactive shell to ask the config location is trustable or not. So we cannot use `mise` binary for this and just use the completion files. I don't think we can avoid this during the installation.

Reply:
ok, env should be setup but AFAIK there is no way to put some other env to `generate_completions_from_executable` and so this is the best I guess.